python/helpers/openai_auth.py

```
@@ -0,0 +1,358 @@
+
+
+def _decode_jwt_payload(token: str) -> dict[str, Any] | None:
+    try:
+        parts = token.split(".")
+        if len(parts) != 3:
+            return None
+        payload = parts[1]
+        payload += "=" * (-len(payload) % 4)
+        decoded = base64.urlsafe_b64decode(payload.encode()).decode("utf-8")
+        return json.loads(decoded)
+    except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)